.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on:
+  - pull_request
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest requests_mock
```

Comment:
I would recommend to use poetry to install the packages that will be tested. Since these dependencies are defined in `pyproject.toml` this shouldn't be necessary (This should also fix the current error that `core` is missing).
Maybe look at this or similar ones: https://github.com/snok/install-poetry